src/displayBackend/drm/Connector.hpp

```diff
@@ -40,16 +38,20 @@ class Display;
  * Provides DRM connector functionality.
  * @ingroup drm
  ******************************************************************************/
-class Connector : public DisplayItf::Connector
+class Connector : public ConnectorBase
```

Comment:
ConnectorBase inherits Connector , please recheck ?

Reply:
it inherits the interface: `DisplayItf::Connector`